.github/workflows/build-past-ci-docker-images.yml

```diff
@@ -24,6 +24,17 @@ jobs:
       -
         name: Check out code
         uses: actions/checkout@v3
+      -
+        id: get-base-image
+        name: Get Base Image
+        env:
+          framework_version: ${{ matrix.version }}
+        run: |
+          echo "base_image=$(python3 -c 'import os; from utils.past_ci_versions import past_versions_testing; base_image = past_versions_testing["pytorch"][os.environ["framework_version"]]["base_image"]; print(base_image)')" >> $GITHUB_OUTPUT
+      -
```

Comment:
past CI docker image building is a bit special - different base docker images are required for different torch/tensorflow versions.